gradle.properties

```diff
@@ -21,8 +21,8 @@ shadowGradlePlugin=2.0.3
 slf4jApiVersion=1.7.25
 
 # Version and base tags can be overridden at build time.
-connectorVersion=0.7.0-SNAPSHOT
-pravegaVersion=0.7.0-50.d513b90-SNAPSHOT
+connectorVersion=0.7.0
+pravegaVersion=0.7.0
```

Comment:
I'm confused about this change. If you see how @vijikarthi used to do it, we did not change that version of Pravega. I thought that was because of the use of Pravega as submodule. Could you clarify that this is really the right thing to do?

See this past commit: https://github.com/pravega/hadoop-connectors/commit/cdffb02045e55bd5be1b18813474d37ef12cfaa6

Reply:
Yes, it's a mistake. I should first create a release branch `r0.7` and then target this PR to `r0.7` branch.